tests/network/bindingplugin.go

```diff
@@ -100,6 +103,75 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(vmi.Status.Interfaces[0].MAC).To(Equal(macAddress))
 		})
 	})
+
+	Context("macvtap", func() {
+		const (
+			macvtapNetworkConfNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
+			macvtapBindingName    = "macvtap"
+		)
+
+		var virtClient kubecli.KubevirtClient
+		var macvtapLowerDevice string
+		var macvtapNetworkName string
+
+		createMacvtapNetworkAttachmentDefinition := func(namespace, networkName, macvtapLowerDevice string) error {
+			macvtapNad := fmt.Sprintf(macvtapNetworkConfNAD, networkName, namespace, macvtapLowerDevice, networkName)
+			return createNetworkAttachmentDefinition(virtClient, networkName, namespace, macvtapNad)
+		}
+
+		BeforeEach(func() {
+			virtClient = kubevirt.Client()
```

Comment:
I think `kubevirt.Client()` it can be called directly, its implemented as singleton and its safe to call it more then once.